app/views/shared/_office_location_fields.html.erb

```diff
@@ -44,14 +44,19 @@
           </div>
 
           <div class="row row-form-wrapper no-buffer">
-              <div class="col-md-4 col-sm-4 col-xs-12 form-group form-group-lg no-pd">
+              <div class="col-md-3 col-sm-3 col-xs-12 form-group form-group-lg no-pd">
                 <%= address.text_field :city, class: "floatlabel form-control interaction-field-control-office-location-address-city", required: true, placeholder: 'CITY' %>
               </div>
-              <div class="col-md-4 col-sm-4 col-xs-12 form-group form-group-lg no-pd">
+              <div class="col-md-3 col-sm-3 col-xs-12 form-group form-group-lg no-pd">
                 <%= address.select(:state, options_for_select(State::NAME_IDS.map(&:last), selected: address.object.state), {include_blank: "SELECT STATE"}, required: true) %>
               </div>
-              <div class="col-md-4 col-sm-4 col-xs-12 form-group form-group-lg no-pd">
-                <%= address.text_field :zip, class: "floatlabel form-control zip interaction-field-control-office-location-address-zip", required: true, placeholder: 'ZIP', oninvalid:"checkZip(this);", oninput:"checkZip(this);" %>
+              <div class="col-md-3 col-sm-3 col-xs-12 form-group form-group-lg no-pd">
+                 <%= address.text_field :zip, class: "floatlabel form-control zip interaction-field-control-office-location-address-zip", required: true, placeholder: 'ZIP',oninvalid:"checkZip(this);", oninput:"checkZip(this);",
+                     onchange: "getCounties(this, \""+f.index.to_s+"\")" %>
```

Comment:
Let's fix this - we don't want to check for counties on EVERY change. We'd be making calls to the server for "0", then "01", then "011", and so on.

We already have a checkZip method, so maybe within that we can dispatch for counties if zip looks okay.